src/tactic/sort.lean

```diff
@@ -0,0 +1,156 @@
+/-
+Copyright (c) 2022 Arthur Paulino, Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Arthur Paulino, Damiano Testa
+-/
+import data.polynomial.degree.definitions
+
+/-!  # A tactic for sorting sums  -/
+
+namespace tactic
+
+/--  Given an expression `ei` and a list `l : list expr`, returns a `tactic expr` that is
+`ei + sum_of_list l`. -/
+meta def build_sum : expr → list expr → tactic expr
```

Comment:
I would guess this has a one-line implementation with `list.mfold`

Reply:
It does!  Thanks for the suggestion!

Reply:
I removed this definition and inlined it in the only place where it was used.